### PR TITLE
Pagination styles

### PR DESCRIPTION
**Pull request by olikjh**
image

### CODE REVIEW

File: lib/spoon-ui.scss [hunk 1]
```diff
@@ -773,21 +773,23 @@ table.table {
 }
 
 ul.pagination {
-  color: rgba(53, 64, 82, 0.5);
+  color: #001c44;
```
Comment (AnnaGuzniak) [inline lib/spoon-ui.scss]: please, could you use this: var(--gray-darker);

File: lib/spoon-ui.scss [hunk 2]
```diff
@@ -773,21 +773,23 @@ table.table {
...
 
   .page-link {
-    color: rgba(53, 64, 82, 0.5);
+    color: rgba(0, 28, 68, 0.5);
     border: none;
```
Comment (AnnaGuzniak) [inline lib/spoon-ui.scss]: may be add, please, var for this color

File: lib/spoon-ui.scss [hunk 3]
```diff
@@ -805,9 +807,8 @@ ul.pagination {
 
   .page-item.active .page-link {
     background-color: unset;
-    color: var(--black-text);
+    color: rgb(0, 28, 68);
     border-color: var(--pagination-border-color);
```
Comment (AnnaGuzniak) [inline lib/spoon-ui.scss]: the same

File: lib/spoon-ui.scss (hunk 1 above)
Comment (olikjh) [reply lib/spoon-ui.scss]: Done. Have not looked into color definitions.

File: lib/spoon-ui.scss (hunk 2 above)
Comment (olikjh) [reply lib/spoon-ui.scss]: It was in the colors already, `--gray-darker-opacity`. Fixed

**Pull request merged**